server/devSetup/README.md

```diff
@@ -0,0 +1,62 @@
+# Getting a Local Server Instance up.
```

Comment:
this looks great! thanks for putting it all together. we could even put this under server/README.md